android/app/build.gradle

```
@@ -48,7 +48,7 @@ android {
         applicationId "com.example.front_end"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
-        minSdkVersion 21
+        minSdkVersion 23
```

```suggestion
        minSdkVersion 21
```
Dont need to upgrade sdkVersion

---

The minimum version of the SDK to run the compass is 23.